human-spaceflight/data/KOCH_CH.md

```diff
@@ -15,11 +15,21 @@ country:
   - USA
 missions:
   - name: Soyuz MS-12
-    time_start:	2019-03-14T19:14Z
-    time_end:	
+    time_start: 2019-03-14T19:14Z
+    time_end:   2020-02-06T00:00Z
 evas:
   - time_start: 2019-03-29T11:42Z
     time_end:   2019-03-29T18:27Z
+  - time_start: 2019-10-06T11:39Z
+    time_end:   2019-10-06T18:40Z
+  - time_start: 2019-10-11T11:38Z
+    time_end:   2019-10-11T18:23Z
+  - time_start: 2019-10-11T11:38Z
+    time_end:   2019-10-18T18:55Z
+  - time_start: 2020-01-15T11:25Z
```

Comment:
Should be - time_start: 2020-01-15T11:35Z ( http://www.spacefacts.de/iss/english/exp_61.htm )